packages/components/src/FormatFile/FormatFile.mdx

```diff
@@ -0,0 +1,60 @@
+---
+name: FormatFile
+menu: Components
+route: /components/format-file
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { FormatFile } from ".";
+import { Content } from "../Content/";
+
+# File Card
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+Pass this a logical description of a file (an object) and it will represent a
+file in a number of lifecycle states.
+
+A progress indicator can be used to display upload status, and a thumbnail or
+icon will be displayed depending on what is provided.
+
+A delete callback function can be passed in, and will be called when the user
+actions the delete button.
+
+```ts
+import { FormatFile } from "@jobber/components/FormatFile";
+```
+
+<Playground>
+  <Content>
+    <FormatFile
+      file={{
+        key: "abc",
+        name: "myballisbigandroundIamrollingitontheground.png",
+        type: "image/png",
+        size: 213402324,
+        progress: 0.3,
+        src: () =>
+          Promise.resolve(
+            "https://res-1.cloudinary.com/crunchbase-production/image/upload/c_thumb,h_256,w_256,f_auto,g_faces,z_0.7,q_auto:eco/ywedismk5vf5uqc2nngg"
+          )
+      }}
```

Comment:
We could pull this out of the playground here and document it separately perhaps? Just to keep this example simpler? If you think this makes sense I can do a quick share and show you how.

Reply:
Which part? I think its actually useful to have this structure laid out here in the example doc.

Reply:
Why don't we talk in the morning? I think it may be easier?